packages/block-editor/src/components/link-control/style.scss

```diff
@@ -0,0 +1,22 @@
+.link-control__popover-inner {
```

Comment:
There are some rules for naming CSS classes after components:

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#css

There is also this related question, which package should have this component.

Reply:
I've fixed this @gziolo. Thanks for the prompt.

There seems to be a consensus that this should live in `block-editor` for now.

Reply:
Okay, I totally missed that there are some dependencies that live in the `block-editor` package. It makes sense to keep it colocated for now. There is another question, whether it should be moved to `components` package at some point but you shouldn't worry about it 😄

Reply:
I still don't see any new commit added which changes class names here. They should all be prefixed with `block-editor-`.